### PR TITLE
material-ui 1.0 definitions

### PR DESCRIPTION
**Pull request by bugzpodder**
Add NativeSelect
Standardize children: React$Node
Standardize Events: SyntheticEvent<*>
Fix new flattened imports

### CODE REVIEW

**Comment by bugzpodder** [comment]
oops did not see https://github.com/flowtype/flow-typed/pull/2249
@KevinGrandon @TrySound

**Comment by KevinGrandon** [comment]
My PR is still lacking I believe, so happy to close that one and go with this. Nice work!

**Comment by KevinGrandon** [comment]
Let's go with 2255 instead, more activity over there. Please close this.

File: definitions/npm/@material-ui/core_v1.x.x/test_core_v1.x.x.js [hunk 1]
```diff
@@ -3,10 +3,10 @@
 import React from "react";
 
 // Direct module import.
-import AppBar from "material-ui/AppBar/AppBar";
+import AppBar from "@material-ui/core/AppBar/AppBar";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_core_v1.x.x.js]:
```js
import AppBar from "@material-ui/core/AppBar";
```

**Comment by bugzpodder** [comment]
@AndrewSouthpaw I dont' understand, can you point me to an example?

**Comment by doberkofler** [comment]
@AndrewSouthpaw I would really need the new @material-ui/core definitions. Any chance to get this changes merged?

**Comment by bugzpodder** [comment]
you can just download a copy of the file and put it in your flow-typed code for now.

**Comment by doberkofler** [comment]
@bugzpodder Thank you. Already done so.

**Comment by hmeerlo** [comment]
Shouldn't the icons be in a seperate definition since they are seperate modules now?

File: definitions/npm/@material-ui/core_v1.x.x/flow_v0.58.x-/core_v1.x.x.js [hunk 2]
```diff
@@ -1594,7 +1741,7 @@ declare module "material-ui/styles/colorManipulator" {
   };
```
Comment (jamessharp) [inline definitions/npm/@material-ui/core_v1.x.x/flow_v0.58.x-/core_v1.x.x.js]: Been around for a while but there's a typo on the line above (can't seem to add a comment in there...)

'ligthen' -> 'lighten'.

Thanks for the great definitions

**Comment by bugzpodder** [comment]
@GAntoine @AndrewSouthpaw  why is this PR being left unattended?

**Comment by GAntoine** [comment]
No reason, just had to get around to it. LGTM and others in this thread, so merging it in.

**Pull request merged**